WowCharComparerWebApp/Logic/Character/Statistics/MainStatsPercentFormater.cs

```diff
@@ -16,6 +16,11 @@ public static string AddMinusToPrimaryStatPercent(string TextToAdd)
                 localText = "-" + TextToAdd;
             }
 
+            if(TextToAdd.Equals("0"))
```

Comment:
To replace whole method you can just simply go for:
`return TextToAdd.Equals("0") ? TextToAdd + "%" : "-" + TextToAdd + "%";`
And use camel case in method's argument!